frontends/infinite-corridor/src/components/LearningResourceCard.test.tsx

```diff
@@ -0,0 +1,51 @@
+import React from "react"
+import { mount } from "enzyme"
+import { LearningResourceCard } from "./LearningResourceCard"
+
+import { makeSearchResult } from "../factories/search"
+import {
+  searchResultToLearningResource,
+  readableLearningResources
+} from "ol-search-ui"
+
+describe("LearningResourceCard component", () => {
+  const render = object => mount(<LearningResourceCard object={object} />)
+  ;["course", "program"].forEach(resourceType => {
```

Comment:
We should use `test.each` for this (or `it.each`). There's a few supported syntaxes... I like:
```
it.each([
  { resourceType: "course" },
  { resourceType: "program" },
])("should render the things we expect for a $resoureType", () => {...})
```